libyara/modules/pe.c

```diff
@@ -1416,6 +1420,10 @@ void pe_parse_header(
       OptionalHeader(pe, DllCharacteristics),
       pe->object, "dll_characteristics");
 
+  directory = pe_get_directory_entry(pe, IMAGE_DIRECTORY_ENTRY_EXPORT);
+  set_integer((directory && directory->VirtualAddress != 0) ? 1 : 0,
+     pe->object, "has_exports");
```

Comment:
Instead of defining `has_exports` I would prefer fixing issue #840 and using `pe.number_of_exports > 0` for checking if the PE has exports.